loader/wsi.c

```diff
@@ -2429,7 +2437,7 @@ bool wsi_swapchain_instance_gpa(struct loader_instance *ptr_instance, const char
     }
 
 #ifdef VK_USE_PLATFORM_WIN32_KHR
-
+#ifndef VULKANSC
```

Comment:
You define the ifndef inside the use platform win32, but then the #endif outside of the XLIB.  This could cause issues and needs to be fixed.

Reply:
fixed